tests/agent_features/test_distributed_tracing.py

```diff
@@ -878,8 +880,7 @@ def _test():
     "full_granularity_enabled,full_granularity_remote_parent_sampled_setting,partial_granularity_enabled,partial_granularity_remote_parent_sampled_setting,expected_sampled,expected_priority,expected_adaptive_sampling_algo_called",
     (
         (True, "always_off", True, "adaptive", None, None, True),  # Uses adaptive sampling algo.
-        (True, "always_on", True, "adaptive", True, 2, False),  # Uses adaptive sampling algo.
-        (False, "always_on", False, "adaptive", False, 0, False),  # Uses adaptive sampling algo.
```

Comment:
I moved this into the new test case below so we can actually check the value of priority.